book/src/installation-binaries.md

```diff
@@ -29,15 +29,19 @@ a portable `x86_64` binary.
 
 1. Go to the [Releases](https://github.com/sigp/lighthouse/releases) page and
    select the latest release.
-1. Download the `lighthouse-${VERSION}-x86_64-unknown-linux-gnu-portable.tar.gz` binary.
-1. Extract the archive:
-    1. `cd Downloads`
-    1. `tar -xvf lighthouse-${VERSION}-x86_64-unknown-linux-gnu.tar.gz`
+1. Download the `lighthouse-${VERSION}-x86_64-unknown-linux-gnu.tar.gz` binary. For example, to obtain the binary file for v4.0.1 (the latest version at the time of writing), a user can run the following commands in a linux terminal:
```

Comment:
The text above talks about downloading a `portable` binary, we should either change the instructions to download portable, or change the text. I think changing the text is probably better